src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -16302,50 +16299,49 @@ void CollectPeakZoneConditions(
             }
 
             // Outside Dry Bulb Temperature
-            compLoad.outsideDryBulb = CalcFinalZoneSizing(zoneIndex).CoolOutTempSeq(timeOfMax);
+            compLoad.outsideDryBulb = calcFinalZoneSizing.CoolOutTempSeq(timeOfMax);
 
             // Outside Wet Bulb Temperature
-            // use standard sea level air pressure because air pressure is not tracked with sizing data
-            if (CalcFinalZoneSizing(zoneIndex).CoolOutHumRatSeq(timeOfMax) < 1.0 &&
-                CalcFinalZoneSizing(zoneIndex).CoolOutHumRatSeq(timeOfMax) > 0.0) {
-                compLoad.outsideWetBulb = PsyTwbFnTdbWPb(state,
-                                                         CalcFinalZoneSizing(zoneIndex).CoolOutTempSeq(timeOfMax),
-                                                         CalcFinalZoneSizing(zoneIndex).CoolOutHumRatSeq(timeOfMax),
-                                                         101325.0);
+            // use standard air pressure because air pressure is not tracked with sizing data
+            if (calcFinalZoneSizing.CoolOutHumRatSeq(timeOfMax) < 1.0 && calcFinalZoneSizing.CoolOutHumRatSeq(timeOfMax) > 0.0) {
+                compLoad.outsideWetBulb = Psychrometrics::PsyTwbFnTdbWPb(state,
+                                                                         calcFinalZoneSizing.CoolOutTempSeq(timeOfMax),
+                                                                         calcFinalZoneSizing.CoolOutHumRatSeq(timeOfMax),
+                                                                         state.dataEnvrn->StdBaroPress);
```

Comment:
Issue correction to use StdBaroPress for moist air property calculations (4 places).